core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -870,5 +921,8 @@ class CivilizationInfoPreview {
 }
 
 enum class CivFlags {
-    cityStateGreatPersonGift
+    cityStateGreatPersonGift, // This should be capitalized, but doing so would break existing save files
```

Comment:
I think we can change this and they'll be fine. We can string compare with ignore case if you want to not miss anything.